src/Common/Handlers/NewEnvironmentHandler.cs

```diff
@@ -52,15 +53,19 @@ private string GetTargetValue(string targetInput)
             string deploymentPackageName,
             string target,
             DirectoryInfo outputDir,
-            string[] replaceTokens
+            string[] replaceTokens,
+            bool interactive
         )
         {
             await EnsureLogin();
 
             name = string.IsNullOrWhiteSpace(name) ? $"Deployment-{Guid.NewGuid()}" : name;
-            string rawParameters = File.ReadAllText(parameters.FullName);
-
-            rawParameters = await Utils.ReplaceTokens(Session, rawParameters, replaceTokens, true);
+            string rawParameters = null;
+            
+            if (parameters != null) {
+                rawParameters = File.ReadAllText(parameters.FullName);
+                rawParameters = await Utils.ReplaceTokens(Session, rawParameters, replaceTokens, true);
+            }

```

Comment:
**parameters** is a mandatory field on both Command and Powershell, this should not be necessary

Reply:
Actually, it's now optional on Powershell but not on CMD. Any reason for that change?

Reply:
Answer to myself: **parameters** are no longer mandatory, they may be set using the **interactive** session